README.md

```diff
@@ -391,6 +396,29 @@ Additional tags/stereotypes can be added to the existing element stereotypes (co
 Each element can be extended with one or multiple custom tags via the keyword argument `$tags="..."`, like `Container(spaAdmin, "Admin SPA", $tags="v1.1")`.
 Multiple tags can be combined with `+`, like `Container(api, "API", $tags="v1.0+v1.1")`.
 
+**Element specific tag definitions**
+
+Sometimes an added element tag is element specific and all element specific colors should be used, e.g. a specific user role should be defined as element tag with the specific colors `...PERSON_...` like
+```csharp
```

Comment:
Should this be `plantuml` rather than `csharp`?

Reply:
fixed